vanetza/asn1/its/DeltaPositions.c

```diff
@@ -15,7 +15,7 @@ static asn_oer_constraints_t asn_OER_type_DeltaPositions_constr_1 CC_NOTUSED = {
 #if !defined(ASN_DISABLE_UPER_SUPPORT) || !defined(ASN_DISABLE_APER_SUPPORT)
 asn_per_constraints_t asn_PER_type_DeltaPositions_constr_1 CC_NOTUSED = {
 	{ APC_UNCONSTRAINED,	-1, -1,  0,  0 },
-	{ APC_CONSTRAINED | APC_EXTENSIBLE,  7,  7,  1,  100 }	/* (SIZE(1..100,...)) */,
+	{ APC_CONSTRAINED | APC_EXTENSIBLE,  5,  5,  1,  32 }	/* (SIZE(1..100,...)) */,
```

Comment:
We should not patch the generated sources but update the used ISO 19321 ASN.1 file which includes the changed sequence constraints. At the moment, we fetch "ed-2" in fetch_iso_asn1_files.cmake, the changed constraints are included in "ed-3".